src/features/island/buildings/Tutorial.tsx

```diff
@@ -55,24 +43,26 @@ export const Tutorial: React.FC<Props> = ({ onClose }) => {
           </p>
         </div>
         <div className="flex space-x-1">
-          <Button onClick={() => setPageNumber(1)}>Back</Button>
+          <Button onClick={() => setPageNumber(0)}>Back</Button>
           <Button onClick={onClose}>Got it</Button>
         </div>
       </>
     );
   };
 
   return (
-    <CloseButtonPanel
-      onClose={onClose}
-      title="Welcome to Buildings"
-      bumpkinParts={bumpkinParts}
-    >
-      <div className="w-full mb-3 flex justify-center">
-        <img src={SUNNYSIDE.icons.hammer} className="h-6 md:h-7" />
+    <>
+      <div className="w-full flex justify-center">
+        <img
```

Comment:
I don't think this image is needed here. I think we can just have the text since we already have the icon in the tab.

Reply:
I removed the image.